README.md

```diff
@@ -1,8 +1,35 @@
 # local-storage-operator
 Operator for local storage
 
+```mermaid
+graph LR
+    PVC -->|Requests local storage using a StorageClass| StorageClass
+    StorageClass -->|Defines a LocalVolume| LocalVolume
+    LocalVolume -->|Represents a physical storage device already mounted on a node| Node
+    Pod -->|References a PVC| PVC
+    PV -->|References| LocalVolume
+    StorageClass -->|Creates| PV
+    VolumeAttachment -->|Correlates| Node
+    VolumeAttachment -->|Correlates| PV
+    LSO((Local Storage Operator)) -->|Manages| LocalVolume
+    LSO((Local Storage Operator)) -->|Manages| StorageClass
+    LocalVolumeSet -->|Discovers local storage devices on node| Node
+    LocalVolumeSet -->|Automatically manages LocalVolume objects|LocalVolume
```

Comment:
LocalVolumeSet does not manages LocalVolumes, it directly creates PVs.

Reply:
Changed flow so that LVS creates PVs https://github.com/openshift/local-storage-operator/commit/f8acb21ec74b51b9fdfde6ab9f8c0a8d71ad3ea9#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R13